### ISSUE TITLE
Does polib compile into mo wihout "msgfmt"?

### ISSUE THREAD

**Issue by buhtz** [izimobil/polib#149]
Do I understand [your code](https://github.com/izimobil/polib/blob/8adc7f9b252eb6653304f8b6f364b119b277e205/polib.py#L550) correct, that when compiling po to mo usig polib that "msgfmt" is not involved?

Polib does the compilation by itself without using external tools or packages?

**Comment by izimobil** [comment]
Polib is a pure python library, of course compiling into MO files is done
in python

**Comment by izimobil** [comment]
By the way please use stackoverflow or what you want for these kind of
questions but not the bug tracker! :)

**Comment by buhtz** [comment]
> By the way please use stackoverflow or what you want
It is a maintainer related question. The bug tracker seems to be the only channel you offering. Direct email is not an option of course.

Maybe you can activate the "Discussion" section here at Microsoft GitHub or create a mailing list (e.g. at python.org).

**Comment by izimobil** [comment]
No time for that, a simple look at the source code would have answered your
question, regards!

**Comment by buhtz** [comment]
> No time for that, a simple look at the source code would have answered your question

My initial post might not be concrete enough. But in my understanding I indicated that I have read your code. I formulated that in my first sentence and also posted a link to the exact method I was looking into.

But it is foreign code and I was not sure if my interpretation is correct.
I just wanted to confirm my results by a yes or no of the maintainer. That's all.
